chart/templates/server-deployment.yaml

```
@@ -151,7 +151,7 @@ spec:
         - name: MINIO_END_POINT
           value: minio.{{ .Release.Namespace }}.svc.cluster.local
         - name: MINIO_PORT
-          value: "9000"
+          value: {{ .Values.minio.port | default "9000" }}
```

This way, if in the values users configure `9000` instead of `"9000"` things will still work.

```suggestion
          value: {{ .Values.minio.port | default 9000 | quote }}
```

---

Alright, I'll make the changes.